package.json

```diff
@@ -47,20 +47,21 @@
   },
   "homepage": "https://github.com/trussworks/react-uswds#readme",
   "peerDependencies": {
+    "@uswds/uswds": "3.1.0",
     "react": "^16.x || ^17.x || ^18.x",
-    "react-dom": "^16.x || ^17.x || ^18.x",
-    "@uswds/uswds": "3.1.0"
```

Comment:
Are we removing the `@uswds/uswds` peer dependency here?

Reply:
Not removed, it was alphabetized by my editor and is now the first one, which seems reasonable.